### ISSUE TITLE
Feature request: Resync for Send Only mode, acknowledge changes from other devices

### ISSUE THREAD

**Issue by eteropczyk** [syncthing/syncthing-android#1685]
Hi Guys, 
first of all great work! I was looking for app good as this!

Case setup:
Mobile device (A) with folder sync to other device (B) running on "Send Only" mode.

Case:
Once file is removed/changed from "B" device, I see warning on "A" device - not synchronized and button appears "Override changes".

I am missing an option to acknowledge these changes - I would like to be able to click a button and sync from B to A. It is the same as if I would change mode from Send Only to "Send & Receive"  before I made changes on B device and switch it back to Send Only once it's done. It would be greate to just click acknowledge changes button instead.

The reason for this is that I clean up my photos on the computer much faster, mainly because they can be previewed better, and then conditionally I would like to transfer these changes to my mobile device. So if its in Send Only mode it should be protected from changes on other devices but I would like to have an option to made those changes if I want to.

Cheers
Piotr

**Comment by imsodin** [comment]
Duplicate of https://github.com/syncthing/syncthing/issues/2710
